_posts/2020-07-12-ray-tune.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title: Dask-ML offers modern hyperparameter optimization techniques packaged in a Scikit-learn API
+tagline: This has been implemented in Dask-ML for over a year
+author: <a href="https://stsievert.com">Scott Sievert</a> (University of Wisconsin–Madison)
+tags: [machine-learning, dask-ml, dask, ray]
+theme: twitter
+---
+
+{% include JB/setup %}
+
+Hyperparameter optimization is the process of deducing machine learning model
+parameters that can't learned from data. These "hyperparameters" have to be
+given at model initialization. An example is with the `alpha` parameter in
+Scikit-learn's [Ridge]. For a more complete overview, see "[How to Use t-SNE
+Effectively][tsne]" or "[Tuning the hyper-parameters of an estimator][tuning]."
+
+[The introduction][1] of Ray's Scikit-learn compatible hyperparameter
+optimization library made an counterfactual statement:
```

Comment:
typo: "an"

And is "counterfactual" the right word here?

Reply:
No, counterfactual is not the right word. I used "exaggerated" instead. Other choices include "inflated" and "disorted".